packages/kit/src/exports/node/index.js

```diff
@@ -59,7 +56,7 @@ function get_raw_body(req, body_size_limit) {
 
 				size += chunk.length;
 				if (size > length) {
-					controller.error(new Error('content-length exceeded'));
+					req.destroy(new Error('request body size exceeded'));
```

Comment:
you could probably do something like check whether `content-length` or `body_size_limit` is smaller outside this loop and provide an appropriate error message here which is more detailed as to which one was exceeded